eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -106,7 +106,7 @@ extends:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-gcc15-amd64
 
       linux_x64_llvmaot:
```

Comment:
Remove usages of this image now that we don't have a special one?

Reply:
Can you share that context at https://github.com/dotnet/runtime/issues/115786 @jkoritzinsky?